components/collector/src/source_collectors/file_source_collectors/generic.py

```diff
@@ -0,0 +1,23 @@
+"""Generic metrics collector."""
+
+from base_collectors import JSONFileSourceCollector
+from collector_utilities.functions import md5_hash
+from source_model import Entity, SourceMeasurement, SourceResponses
+
+
+class GenericSecurityWarnings(JSONFileSourceCollector):
+    """Generic collector for security warnings."""
+
+    async def _parse_source_responses(self, responses: SourceResponses) -> SourceMeasurement: # skipcq: PY-D0003
```

Comment:
```suggestion
    async def _parse_source_responses(self, responses: SourceResponses) -> SourceMeasurement:  # skipcq: PY-D0003
```

Reply:
We use two spaces before inline comments. Hopefully the line doesn't become too long because of this :-)